experiment/measurer/measure_manager.py

```diff
@@ -494,24 +495,84 @@ def extract_corpus(self, corpus_archive_path) -> bool:
         extract_corpus(corpus_archive_path, unit_blacklist, self.corpus_dir)
         return True
 
-    def archive_crashes(self, cycle):
-        """Archive this cycle's crashes into filestore."""
-        if not os.listdir(self.crashes_dir):
-            logs.info('No crashes found for cycle %d.', cycle)
+    def save_crash_metadata(self, crashes, cycle):
+        """Stores crash metadata in database and per-cycle metadata file."""
+        if not crashes:
             return
 
-        logs.info('Archiving crashes for cycle %d.', cycle)
+        logs.info('Storing crashes in database.', crashes)
```

Comment:
I'm surprised this doesn't throw in error.

Reply:
Fixed, it did not.